run.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+# Helper script to create recipes for CRAN R packages to submit to conda-forge.
+#
+# Setup checklist:
+#
+# 1. Put the name of the package(s) in packages.txt
+# 2. Add your GitHub username to the list of maintainers in extra.yaml
+# 3. Run `python run.py` in this directory
+# 4. Move the recipe directory to staged-recipes/recipes
+#
+# Installation requirements:
+#
+# conda, conda-build 2, python 3
```

Comment:
conda-build 3 I guess

Reply:
Not yet. This PR is to replace the current functionality with a Python script. We'll need to make some decisions about what to include from the cb3 cran skeleton, so I'll send a follow-up PR for discussion after this one is merged.